docs/introduction.md

```diff
@@ -91,4 +91,6 @@ Apache InLong serves the entire life cycle from data collection to landing,  and
 |              | Greenplum         | 4.x, 5.x, 6.x                | Lightweight, Standard |
 |              | Elasticsearch     | 6.x, 7.x                     | Lightweight, Standard |
 |              | SQLServer         | 2012, 2014, 2016, 2017, 2019 | Lightweight, Standard |
+|              | Doris             | >= 0.13                      | Lightweight, Standard |
+|              | StarRocks         | >= 2.0                       | Lightweight, Standard |
```

Comment:
please update the related Chinese document at the same time.

https://github.com/apache/inlong-website/blob/master/i18n/zh-CN/docusaurus-plugin-content-docs/current/introduction.md